pkg/tsdb/cloudwatch/types.go

```diff
@@ -0,0 +1,16 @@
+package cloudwatch
+
+import (
+	"github.com/aws/aws-sdk-go/service/cloudwatch"
+)
+
+type CloudWatchQuery struct {
```

Comment:
I think it would be better to include the type in this struct and parse it all at once.

Reply:
I don't include the type, because there are many query type, and I don't want to prepare `struct` for the each query.